perllib/Open311/PostServiceRequestUpdates.pm

```diff
@@ -58,6 +59,10 @@ sub open311_params {
         $open311_conf{use_customer_reference} = 1;
     }
 
+    if ( $body->areas->{+COUNCIL_ID_BUCKS} ) {
+        $open311_conf{mark_reopen} = 1;
+    }
```

Comment:
I would prefer this if it was in the DB, especially as there's a chance of using this with OCC.